src/sage/algebras/quatalg/quaternion_algebra.py

```diff
@@ -1758,9 +1759,48 @@ def discriminant(self):
 
         return (MatrixSpace(QQ, 4, 4)(L)).determinant().sqrt()
 
+    def is_maximal(self):
+        r"""
+        Return ``True`` if the order is maximal in the ambient quaternion algebra, ``False`` otherwise
```

Comment:
The first line should be short and usually we start such description with `Check whether...`
```diff
-        Return ``True`` if the order is maximal in the ambient quaternion algebra, ``False`` otherwise
-        Only works in quaternion algebras over number fields
+        Check whether the order of ``self`` is maximal in the ambient
+        quaternion algebra.
+
+        Only works in quaternion algebras over number fields
```